src/main/java/team4/footwithme/member/domain/Member.java

```diff
@@ -74,7 +77,24 @@ public static Member create(String email, String password, String name, String p
             .build();
     }
 
-    public void encodePassword(PasswordEncoder passwordEncoder) {
-        password = passwordEncoder.encode(password);
+
+    public void update(String name, String phoneNumber, Gender gender){
+        if(name != null)
+            this.name = name;
+
+        if(gender != null)
+            this.gender = gender;
+
+        if(phoneNumber != null)
+            this.phoneNumber = phoneNumber;
+    }
```

Comment:
저는 개인적으로 if문 쓸 때 한줄이더라도 명시적으로 괄호르르 하는걸 좋아하긴 합니다.. ㅎㅎ
```suggestion
        if(name != null) {
            this.name = name;
        }
        if(gender != null) {
            this.gender = gender;
        }
        if(phoneNumber != null) {
            this.phoneNumber = phoneNumber;
        }
    }
```

Reply:
아하 참고하겠습니다